subiquity/models/filesystem.py

```diff
@@ -602,7 +690,21 @@ def supported_actions(self):
         return actions
 
     _can_INFO = True
-    _can_PARTITION = property(lambda self: self.free_for_partitions > 0)
+
+    @property
+    def _can_REFORMAT(self):
```

Comment:
reformat is only for reusing existing partitions?

Reply:
I guess there would be no harm in enabling in other cases, indeed. Changed.